react/features/base/config/functions.js

```diff
@@ -177,7 +177,7 @@ export function overrideConfigJSON(
                 logger.info(
                     `Extending ${configName} `
                     + `with: ${JSON.stringify(configJSON)}`);
-                _.merge(configObj, configJSON);
+                _merge(configObj, configJSON);
```

Comment:
Cannot we merely use `_.mergeWith` and `objValue => Array.isArray(objValue) ? objValue : undefined` as its `customizer`?